### ISSUE TITLE
Children are not being watched by adults

### ISSUE THREAD

**Issue by mijkehurk** [lvanhee/COVID-sim#99]
Children at home/school/away are mostly not being watched by adults, i.e. "[is-currently-watched-by-an-adult?] of children" is mostly false, also when they are at home.

**Comment by lvanhee** [comment]
Thanks for warning.
This should not be too abnormal, though some improvements can be made.
 [is-currently-watched-by-an-adult?] should be read "is-currently-watched-by-a-parent?".
A kid should be watched by a parent when not at school. Do you see any abnormal behaviour regarding this? If so, there is a bug to fix; else it is mostly some renaming.

**Comment by mijkehurk** [comment]
Yes, adult should also be changed in parent I guess. Also within the reporter itself, as it is checked by: "report any? [current-gatherers with [is-adult?]] of current activity" so not necessarily the parent.

But still, I found children at home without any adult being there too.

**Comment by lvanhee** [comment]
Ok. Sounds like there is something weird here. Thanks for checking, I keep
this open.
